bookOfHb.md

```diff
@@ -0,0 +1,216 @@
+# The Book of Self-Repair (Heartbeat First Impressions)
+
+
+## As a developer, I want to successfully hack on Self-Repair
+
+### Setup (do once)
+
+1. Pots and Pans
+
+	```
+	# required: git, node
+	brew install node   # includes npm
+	brew install git    # git
+	
+	# optional
+	brew install hub    # github's drop in git client
+	brew install ghi    # github issues clien
+	```
+	
+
+1. Groceries
+
+	```
+	# usual clone and fork
+	
+	git clone mozilla/self-repair-server
+	cd self-repair-server
+	hub fork       
+	```
+
+1. Prep the veggies
+
+	```
+	# usual node package install
+	npm install   
+	# creates dependecies at `node_modules`
+	```
+
+1. Adjust the oven.
+
+	**Firefox Settings** - allow local debug in `about:config`
+	
+	- `browser.uitour.requireSecure;false`
+	- `browser.uitour.testingOrigins;http://localhost`
+
+
+### Modify 
+
+```
+ghi create -m "Some issue that need fixing"  
+git checkout -b 151-my-issue
+
+# edit files in `src/`
+
+```
+
+### Build 
+
+```
+npm run compile  
```

Comment:
When I run `npm run compile`, I get an error about missing webpack:

```
(140-fix=85ec2) saturn2 ~/mozilla/self-repair-server> npm run compile

> self-repair-server@0.11.1 precompile /home/willkg/mozilla/self-repair-server
> command rm -rf deploy && mkdir deploy && node scripts/all-locales.js


> self-repair-server@0.11.1 compile /home/willkg/mozilla/self-repair-server
> webpack

sh: webpack: command not found
npm ERR! weird error 127
npm ERR! not ok code 0
```

Did I miss a step in setup?